content/fr/tracing/setup_overview/setup/dotnet-core.md

```diff
@@ -423,6 +423,7 @@ Le tableau suivant énumère les variables de configuration disponibles aussi bi
 | `DD_AGENT_HOST`                                     | Définit le host vers lequel les traces sont envoyées (le host qui exécute l'Agent). Il peut s'agir d'un hostname ou d'une adresse IP. Ce paramètre est ignoré si `DD_TRACE_AGENT_URL` est défini. Valeur par défaut : `localhost`.                 |
 | `DD_TRACE_AGENT_PORT`                               | Définit le port sur lequel les traces sont envoyées (le port où l'Agent écoute les connexions). Ce paramètre est ignoré si `DD_TRACE_AGENT_URL` est défini. Valeur par défaut : `8126`.                     |
 | `DD_LOGS_INJECTION`<br/><br/>`LogsInjectionEnabled` | Active ou désactive l'injection automatique des identifiants de corrélation dans les logs de l'application.                                                                                               |
+| `DD_TRACE_SAMPLE_RATE`<br/><br/>`GlobalSamplingRate` | Active ou désactive [Tracing without limits][4].                                                                                                                         |
```

Comment:
```suggestion
```
Please remove this line, since it introduces a misnamed link.

In general, you can leave the French and Japanese files alone. The translation process will pick up the English changes, and create updated translations.